iot_api/user_api/resources/endpoints.py

```diff
@@ -27,20 +27,26 @@
 from iot_api.user_api.enums import RoleTypes
 from iot_api.user_api.events.data_collector_events import emit_data_collector_event
 from iot_api.user_api.events.policy_events import emit_policy_event
-from iot_api.user_api.model import User, Organization, Device, RevokedTokenModel, AccountActivation, StatsCounters, \
-    PasswordReset, LoginAttempts, UserRole, UserToUserRole, ChangeEmailRequests, Alert, AlertType, Packet, \
-    SendMailAttempts, GlobalData, \
+from iot_api.user_api.model import User, Organization, Device, AccountActivation, \
+    UserRole, UserToUserRole, Alert, AlertType, Packet, \
     get_user_collector_ids
+from iot_api.user_api.models.ChangeEmailRequests import ChangeEmailRequests
```

Comment:
Same here, the models can be imported directly from models, since they are in the models __init__ file.

Reply:
Done!